RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/constants/Constants.kt

```diff
@@ -5,13 +5,18 @@ import java.net.URL
 
 internal object Constants
 {
-    val apiUrl = URL(BuildConfig.API_URL)
+
+	val apiUrl = URL(BuildConfig.API_URL)
     const val apiVersion = "v3"
 
     const val maxEventSize = 16384L // 16 KB
     const val maxRequestBodySize = 1048576L // 1 MB
     const val maxRequestBodySizeBuffer = 5120L // 5KB
 
-    const val eventDefaultAnalyticsSystemName = "_GENERIC" // todo: set
```

Comment:
po co to todo? to domyslne wartosci i juz

Reply:
Usunąłem todo, było bo nie byłem pewien, czy to nazwa finalna